addons/website/static/src/snippets/s_website_form/000.js

```diff
@@ -228,8 +228,18 @@ odoo.define('website.s_website_form', function (require) {
             // All 'hidden if' fields start with d-none
```

Comment:
Commit message:
- `Following [4], The ...` => nitpick: remove the capital "T" :wink: